app/_hub/kong-inc/aws-lambda/index.md

```diff
@@ -100,7 +100,7 @@ params:
       description: |
         The [`LogType`](http://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_RequestSyntax) to use when invoking the function. By default, `None` and `Tail` are supported.
     - name: timeout
-      required: false
+      required: true
```

Comment:
Reviewers: so the schema.lua file lists these (timeout and keep alive) as required = true, so I changed them. The description however says optional. So are these fields required or optional?

Reply:
My assessment is that the field is required but it is optional to explicitly state in your curl command.